src/pages/pu-settings/Form/PUSettingsForm.js

```diff
@@ -0,0 +1,149 @@
+import { useContext } from 'react'
+import { Grid } from '@mui/material'
+import * as yup from 'yup'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { SystemRepository } from 'src/repositories/SystemRepository'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import ResourceComboBox from 'src/components/Shared/ResourceComboBox'
+import { useResourceQuery } from 'src/hooks/resource'
+import { DataSets } from 'src/resources/DataSets'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { InventoryRepository } from 'src/repositories/InventoryRepository'
+import CustomNumberField from 'src/components/Inputs/CustomNumberField'
+import { useForm } from 'src/hooks/form'
+import FormShell from 'src/components/Shared/FormShell'
+import { ControlContext } from 'src/providers/ControlContext'
+import toast from 'react-hot-toast'
+
+const PUSettingsForm = () => {
+  const { getRequest, postRequest } = useContext(RequestsContext)
```

Comment:
remove getRequest

Reply:
it is needed for getting the data in the first load